parquet-column/src/main/java/org/apache/parquet/column/values/rle/RunLengthBitPackingHybridDecoder.java

```diff
@@ -43,7 +45,7 @@
 
   private final int bitWidth;
   private final BytePacker packer;
-  private final ByteArrayInputStream in;
+  private final ByteBufferInputStream in;
```

Comment:
It looks like this class just depends on InputStream. So let's just use InputStream here.